### PR TITLE
fix urls

### PR DESCRIPTION
**Pull request by jloboescalona2**
this PR apply:
- master

### CODE REVIEW

File: src/public/react/src/constants/literals.js [hunk 1]
```diff
@@ -37,16 +37,16 @@ export const SIZE = 'Size: ';
...
-  '<a href="http://opennebula.systems/">OpenNebula Systems.</a>';
+  '<a href="http://opennebula.io/">OpenNebula Systems.</a>';
 export const FOOTER_TEXT =
-  '© %YEAR% OpenNebula Systems S.L. All Rights Reserved. %LINK1% Please send comments to %LINK2% This site is managed by %LINK3% <br> OpenNebula and OpenNebula Systems are not responsible for any content you may access or download from external sites';
+  '© %YEAR% OpenNebula Systems. All Rights Reserved. %LINK1% Please send comments to %LINK2% This site is managed by %LINK3% <br> OpenNebula and OpenNebula Systems are not responsible for any content you may access or download from external sites';
 export const PRINCIPAL_MENU = [
   {
     title: 'Documentation',
-    url: 'http://marketplace.opennebula.systems/docs/',
+    url: 'http://marketplace.opennebula.io/docs/',
```
Comment (vholer) [inline src/public/react/src/constants/literals.js:49]: Better `https://docs.opennebula.io/appliances/`

Also, ideally all the rest references point directly to https://

Comment (jloboescalona2) [reply src/public/react/src/constants/literals.js]: I added the https and the url as you said, thanks!

**Comment by vholer** [comment]
But, I would say the bundles needs to be refreshed again? @jloboescalona2

**Comment by jloboescalona2** [comment]
@vholer, what do you mean by refresh? if it is the refresh of the browser. Yes, because I suppose you will have the old  in the cache of your browser

**Comment by vholer** [comment]
No, I mean if the JavaScript bundles don't need to be generated again after you did last change in the "literals". Because, I don't see changed URLs to be working, still see the old ones.

**Comment by jloboescalona2** [comment]
yes, sorry you have to replace the bundles files

**Pull request merged**